gnomad_qc/v3/annotations/generate_sv_hists.py

```diff
@@ -0,0 +1,197 @@
+# noqa: D100
+import argparse
+import logging
+
+import hail as hl
+from gnomad.utils.slack import slack_notifications
+
+from gnomad_qc.slack_creds import slack_token
+from gnomad_qc.v3.resources.annotations import sv_age_and_gq_hists
+from gnomad_qc.v3.resources.basics import (
+    gnomad_sv_autosome_vcf_paths,
+    gnomad_sv_release_samples_list_path,
+    gnomad_sv_sex_vcf_paths,
+    temp_gnomad_sv_mt_path,
+)
+from gnomad_qc.v3.resources.meta import meta
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s: %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger("generate_gnomad_sv_histograms")
+logger.setLevel(logging.INFO)
+
+
+def generate_hists(mt: hl.MatrixTable) -> hl.Table:
+    """
+    Generate histograms for age and GQ for gnomAD SVs.
+
+    :param mt: gnomAD SV MatrixTable.
+    :return: Table with histograms.
+    """
+    logger.info("Generating histograms for age and GQ...")
+    hists = mt.select_rows(
+        age_hist_het=hl.or_missing(
+            ~mt.filters.contains("MULTIALLELIC"),
+            hl.agg.filter(mt.GT.is_het(), hl.agg.hist(mt.age, 30, 80, 10)),
+        ),
+        age_hist_hom=hl.or_missing(
+            ~mt.filters.contains("MULTIALLELIC"),
+            hl.agg.filter(mt.GT.is_hom_var(), hl.agg.hist(mt.age, 30, 80, 10)),
+        ),
+        gq_hist_alt=hl.or_missing(
+            ~mt.filters.contains("MULTIALLELIC"),
+            hl.agg.filter(mt.GT.is_non_ref(), hl.agg.hist(mt.GQ, 0, 100, 20)),
+        ),
+        gq_hist_all=hl.or_missing(
+            ~mt.filters.contains("MULTIALLELIC"), hl.agg.hist(mt.GQ, 0, 100, 20)
+        ),
+    ).rows()
+
+    hist_expr = {}
+    for hist in ["age_hist_het", "age_hist_hom", "gq_hist_alt", "gq_hist_all"]:
+        hist_expr.update(
+            {
+                f"{hist}_bin_freq": hl.delimit(hists[hist].bin_freq, delimiter="|"),
+                f"{hist}_bin_edges": hl.delimit(hists[hist].bin_edges, delimiter="|"),
+                f"{hist}_n_smaller": hists[hist].n_smaller,
+                f"{hist}_n_larger": hists[hist].n_larger,
```

Comment:
basic question, do we need these for the GQ hists? it looks like those are all 0's in the test HT you pointed to (`gs://gnomad-tmp-4day/gnomad_v3.1.2_qc_data/gnomad_v3_sv_data/gnomad_v3_sv_temp.hists.ht`)
```
>>> ht.aggregate(hl.agg.stats(ht.gq_hist_alt_n_larger))
Struct(mean=0.0, stdev=0.0, min=0.0, max=0.0, n=33823, sum=0.0)
>>> ht.aggregate(hl.agg.stats(ht.gq_hist_alt_n_smaller))
Struct(mean=0.0, stdev=0.0, min=0.0, max=0.0, n=33823, sum=0.0)
```